DCode.Web/Views/Reporting/TopContributors.cshtml

```diff
@@ -0,0 +1,54 @@
+﻿@model IEnumerable<SearchApplication.Models.Contributor>
+
+@{
+    ViewBag.Title = "Top Contributors List";
+    Layout = "~/Views/Shared/_LayoutReporting.cshtml";
+}
+
+<h2>Top Contributors List</h2>
+
+<form method="post" action="">
+
+    <table>
+        <tr>
+            <th>
+                @Html.ActionLink("Export To Excel", "ExportToExcel", "Reporting")
+            </th>
+        </tr>
+    </table>
+
+</form>
+
+
+<table class="table table-bordered table-responsive table-hover">
```

Comment:
Can we componentize this table in a partial and use it in `TopContributors.cshtml` , `TopRequestors.cshtml` &  `SplitPerOffering.cshtml`.